sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -3909,6 +3943,11 @@ class AstBuilder extends DataTypeAstBuilder with SQLConfHelper with Logging {
     }
   }
 
+  /**
+   * Converts a string literal to a string.
+   */
+  protected def stringLitToStr(x: StringLitContext) = string(visitStringLit(x))
```

Comment:
Is it used multiple times? If not, please, inline it.

Reply:
> this logic is used in couple of place

Where?
```
$ find . -name "*.scala" -print0|xargs -0 grep -n 'stringLitToStr'
./sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala:173:    val queryString = Option(ctx.queryParam.stringLit()).map(sl => Left(stringLitToStr(sl)))
./sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala:3968:  protected def stringLitToStr(x: StringLitContext) = string(visitStringLit(x))
```

Reply:
I meant getting string from StringLit node